installation/single_machine_docker.rst

```diff
@@ -7,15 +7,31 @@ This section describes setting up a Citus cluster on a single machine using dock
 
 **1. Install docker and docker-compose**
 
-Follow the installation instructions for `Docker Engine <https://docs.docker.com/engine/installation/>`_ and `Docker Compose <https://docs.docker.com/compose/install/>`_ on your platform.
+* Install the `Docker Platform <https://www.docker.com/products/overview#/install_the_platform>`_.
+* (Mac only) Start the Docker daemon
+
+  .. code-block:: bash
+
+    # Mac ONLY
+
+    open /Applications/Docker.app
+
+* (Linux only) On most platforms the previous step will install docker-compose as well as Docker, but on Linux you'll need to install docker-compose manually by downloading the binary and putting it in your path:
```

Comment:
Small rewording.

On Linux, you need to manually install docker-compose (you can find the latest docker-compose version [here](link)). For this, you need to download the binary and put it in your path: